src/PluginForm/RazorpayForm.php

```diff
@@ -33,6 +33,15 @@ public function createOrGetRazorpayOrderId($order, $orderData)
     {
         $create = false;
 
+        if ($orderData['currency'] === "KWD" or
```

Comment:
can we put the unsupported currencies in a constant and use the constant to check?